code/__DEFINES/html_assistant.dm

```diff
@@ -0,0 +1,29 @@
+#define TOOLTIP_CSS_SETUP \
+"<style>\
+.tooltip {\
+  position: relative;\
+  display: inline-block;\
+  border-bottom: 2px dotted;\
+}\
+\
+.tooltip .tooltiptext {\
+  visibility: hidden;\
+  background-color: black;\
+  color: #fff;\
+  text-align: center;\
+  border-radius: 3px;\
+  border: 1px solid grey;\
+  padding: 10px 17px;\
+  \
+  position: absolute;\
+  z-index: 1;\
+}\
+\
+.tooltip:hover .tooltiptext {\
+  visibility: visible;\
+}\
+</style>"
+// I tried `width: max-content;` in css, but it's not working. giving custom width_px only works for now.
```

Comment:
IE11 does not support the max-content attribute.

Reply:
that explains why